src/Sulu/Bundle/ContentBundle/Tests/Functional/Controller/NodeControllerTest.php

```diff
@@ -2089,7 +2089,7 @@ public function testInternalLinkAutoName()
     public function testRenamePageWithLinkedChild()
     {
         $client = $this->createAuthenticatedClient();
-        $this->importer->import(__DIR__ . '/../../app/Resources/exports/tree.xml');
+        $this->importer->import(__DIR__ . '/../../files/exports/tree.xml');
```

Comment:
Maybe fixtures instead of files? Or is that a bad idea? :thinking:

Reply:
👍

Reply:
there exist a Tests/Fixtures folder but I think this classes are can be removed as they really seems to be unused and extend from a strange not existing class: https://github.com/sulu/sulu/tree/2.0.0-alpha4/src/Sulu/Bundle/ContentBundle/Tests/Fixtures

Reply:
Can you remove them as well, if they are really unused? :see_no_evil:

Reply:
this are removed